### PR TITLE
Bioref: minimum number of observation statistical queries

### PR DESCRIPTION
**Pull request by romainbou**

### CODE REVIEW

File: src/app/config/config.dev-server.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
-  "medco-node-url": "http://localhost:3100",
+  "medco-node-url": "https://swiss-bioref.chuv.ch/api/",
 
-  "keycloak-url": "http://localhost:8080/auth",
+  "keycloak-url": "https://auth.tuneinsight.com/auth",
   "keycloak-realm": "ti-realm",
-  "keycloak-client-id": "glowing-bear",
+  "keycloak-client-id": "bioref_chuv-gb-s6yhtgriucm8hhtdg9mi",
```
Comment (f-marino) [inline src/app/config/config.dev-server.json:7]: file to be reverted

File: src/app/modules/gb-explore-module/gb-explore.component.html [hunk 2]
```diff
@@ -65,17 +65,24 @@
     <br>
     <p-accordion class="gb-additional-param-wrapper">
       <p-accordionTab header="Additional settings" [selected]="false">
-        <p>Defaults recommended by the CLSI guidelines</p>
-        <div class="gb-additional-param"><label>Number of Bootstrap iterations</label><br>
-          <input type="number" [(ngModel)]="bootR" pInputText placeholder="Number of Bootstrap iterations" />
+        <h5>Defaults recommended by the CLSI guidelines</h5>
```
Comment (f-marino) [inline src/app/modules/gb-explore-module/gb-explore.component.html:68]: in the end this part should be removed/hidden according to what we discussed with Matthias. Maybe showing it only if a dedicate env is set, for test purposes? (only if it's worth it)

File: src/app/services/explore-statistics.service.ts [hunk 3]
```diff
@@ -341,16 +346,20 @@ export class ExploreStatisticsService {
...
 
-                    if (newChartInformation.numberOfObservations() > 0) {
+                    const numberOfObservations = newChartInformation.numberOfObservations();
+                    if (numberOfObservations >= this._minSampleSize) {
                         return [
                             ...responseResult,
                             newChartInformation
                         ];
+                    } else if (numberOfObservations > 0) {
+                        MessageHelper.alert('info', `Number of observations (${numberOfObservations}) insufficient for the Reference Interval. Minimum ${this._minSampleSize} required`);
```
Comment (f-marino) [inline src/app/services/explore-statistics.service.ts:361]:
```suggestion
                        MessageHelper.alert('info', `Number of observations (${numberOfObservations}) insufficient to compute reference interval (minimum ${this._minSampleSize} required`);
```

Comment (romainbou) [reply src/app/services/explore-statistics.service.ts]: Thanks done

File: src/app/config/config.dev-server.json (hunk 1 above)
Comment (romainbou) [reply src/app/config/config.dev-server.json:7]: Done

File: src/app/modules/gb-explore-module/gb-explore.component.html (hunk 2 above)
Comment (romainbou) [reply src/app/modules/gb-explore-module/gb-explore.component.html:68]: Now it's only showing on "Dev" mode

**Comment by romainbou** [comment]
Tested with low number on observation. It works and now redirect back to the explore tab

Screenshot from 2022-11-09 12-40-18

**Pull request merged**